.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "src/fcc"]
 	path = src/fcc
 	url = https://github.com/haddocking/fcc.git
+	ignore = dirty
```

Comment:
once `fccpy` gets developed this should become a dependency and not a submodule.